### DISCUSSION TITLE
Display accounts types in drop-down list of possible source/destination account?

### DISCUSSION THREAD

**Discussion by Ezwen** [firefly-iii/firefly-iii#7746] [Firefly III] [answered]
Hi!

When creating a new deposit/withdrawal/transfer, we must use a drop down list to choose a source account and a destination account. For instance this is what we see when choosing a source account:

![Capture d’écran du 2023-07-11 10-11-02_redacted](https://github.com/firefly-iii/firefly-iii/assets/5868014/959ee8a5-76c0-491d-8862-cdc524e52805)

My question is: **is there any way to make Firefly show us the _type_ (asset, expense, revenue, liability) of each account shown in this list?** In my case where I use lots of accounts, this would make browsing the list much more easier.

Regards and thanks,

**Comment by JC5** [comment, marked as answer]
I'm afraid not, but this is something I have on my list for the new layout.

**Comment by Ezwen** [comment]
I see, thanks for the swift answer! Looking forward to the new layout then :)

**Reply by Ezwen** [reply]
Would it help that I create a formal feature request, to ease tracking?

**Reply by JC5** [reply]
I got it fam 😎
